prov/efa/src/rxr/rxr_msg.c

```diff
@@ -55,157 +55,85 @@
  *  Send function
  */
 
-/**
- *   Utility functions used by both non-tagged and tagged send.
- */
 static inline
-ssize_t rxr_msg_post_cuda_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
+int rxr_msg_select_rtm_for_cuda(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
```

Comment:
Could you add a docstring for this function?

Reply:
Added in new revision.